### PR TITLE
(Chore) amsterdam-stijl Removal helper components

### PR DESCRIPTION
**Pull request by janjaap**
This PR adds components that will replace specific styling, now supplied by the `amsterdam-stijl` package. In a follow-up pull request, these components will be applied.

### CODE REVIEW

File: src/components/Checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { Checkbox as AscCheckbox, themeSpacing } from '@datapunt/asc-ui';
+
+const Wrapper = styled.span`
+  display: inline-block;
+  background: white;
+  padding-right: ${themeSpacing(2)};
+  background-clip: content-box;
+`;
+
+const StyledBox = styled(AscCheckbox)`
```
Comment (aditudorache) [inline src/components/Checkbox/index.js:13]: `CheckboxStyle` would be more readable

File: src/components/ButtonBar/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { themeSpacing } from '@datapunt/asc-ui';
+
+const Bar = styled.div`
```
Comment (aditudorache) [inline src/components/ButtonBar/index.js:6]: `ButtonBarStyle`  would be more readable

File: src/components/Checkbox/index.js (hunk 1 above)
Comment (janjaap) [reply src/components/Checkbox/index.js:13]: But it's not style, it's a component.

File: src/components/ButtonBar/index.js (hunk 2 above)
Comment (janjaap) [reply src/components/ButtonBar/index.js:6]: Same here, the result of a call to `styled` is a component, not a list of styled, which the name `ButtonBarStyle` applies

**Pull request merged**